doc/sphinx/using/libraries/index.rst

```diff
@@ -16,6 +20,23 @@ the `Coq Package Index <https://coq.inria.fr/opam/www/>`_ and
 installed with the `opam package manager
 <https://coq.inria.fr/opam-using.html>`_.
 
+:gdef:`Libraries <library>` contain only compiled Coq scripts.
+:gdef:`plugins <plugin>` can also include compiled OCaml code that can change
+the behavior of Coq.  Both are :term:`packages <package>`.
+While users configure and load them identically, there are a few differences
+to consider:
+
+- Nearly all plugins add things to the vernacular language that could not be
+  added otherwise, so new top-level commands or tactics are likely available.
+- Plugins can do almost anything to Coq. Therefore they have more potential to
+  expand the trusted code base than libraries, so they shouldn't be fully
+  trusted.  If desired, you can reduce the risk by running :ref:`coqchk`
+  on compiled scripts that load plugins.
```

Comment:
"Things" is in my opinion too informal. Also I think some statements are too sweeping here.
```suggestion
- Nearly all plugins add functionality to Coq that could not be added otherwise,
  so they likely provide new top-level commands or tactics.
- Compared to libraries, plugins can change Coq's behavior in many possibly
  unexpected ways. Therefore, using a plugin requires a degree of trust in its authors
  going beyond the trust required in libraries. If desired, you can mitigate trust issues
  by running :ref:`coqchk` on compiled files produced from Coq scripts that load plugins.
```